### ISSUE TITLE
Auth rules wrongly claim that users can demote others of the same power level

### ISSUE THREAD

**Issue by turt2live** [matrix-org/matrix-spec#699]
The auth rules (v1 as a base) only consider a case where the user's power level is *higher*, but *higher* is somewhat ambiguous. It should read 'equal to or greater than'.

For reference, here's the current auth rules:
![image](https://user-images.githubusercontent.com/1190097/94747193-84bf1080-033b-11eb-8548-901fa9c79a31.png)

When deleting an entry (not changing it), 10d triggers but not 10e.

**Comment by richvdh** [comment]
Right, yes. One way to fix this would be to extend 10e to cover "For each entry being changed or removed ... other than the sender's own entry", but I find the whole thing hard to follow. #1269 rephrases the whole thing.
